routes/conversation.js

```diff
@@ -30,7 +30,7 @@ router.post("/", async function (req, res) {
     });
     await newConversation.save();
     return res.status(200).json({
-        message: "Conversation Created",
+        ...newConversation._doc,
```

Comment:
Instead of returning the whole doc, might just be easier to return the document ID  (newConversation._id) since you are returning unnecessary data that the front-end posted